Gemfile.lock

```diff
@@ -1,7 +1,8 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    RedCloth (4.2.9)
+    CFPropertyList (2.3.5)
```

Comment:
I don't think we have strict guidelines as to whether or not you should commit your `Gemfile.lock` with feature PRs, but to me it makes more sense to add it in a separate commit instead. Not a huge deal either way though.